rust/windows-client/src-tauri/src/client/logging.rs

```diff
@@ -101,24 +103,27 @@ pub(crate) async fn export_logs_to(file_path: PathBuf) -> Result<()> {
     Ok(())
 }
 
-#[derive(Clone, Serialize)]
+#[derive(Clone, Default, Serialize)]
 struct FileCount {
-    files: u64,
     bytes: u64,
+    complete: bool,
+    files: u64,
 }
 
 pub(crate) async fn count_logs(app: tauri::AppHandle) -> Result<()> {
     let mut dir = fs::read_dir("logs").await?;
-    let mut files: u64 = 0;
-    let mut bytes: u64 = 0;
+    let mut file_count = FileCount::default();
+    // Zero out the GUI
+    app.emit_all("file_count_progress", &file_count)?;
```

Comment:
I think this keeps growing in complexity and it's not worth it having a "live" count of the number of files/bytes.

I'd propose we just immediately return here with the number of files.

Reply:
Okay. And just show like a "Counting..." while it counts?

Reply:
@ReactorScram Yeah, "Calculating..." sounds good.